### PR TITLE
Skip unsupported api

### PR DESCRIPTION
**Pull request by alonbrody**
(Asana)[https://app.asana.com/0/280843276935180/272520507930807]

This PR introduces skipping unsupported metrics (similar to skipping missing data error)

@alonweissfeld Thanks

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -329,14 +330,16 @@ FacebookInsightStream.prototype.handleError = function ( error, retry ) {
...
 function SkippedError ( error ) {
     return error.skip === true;
 }
 
 function errorHandler ( options, body )  {
     if ( body.error ) {
-        body.error.skip = options.ignoreMissing && body.error.code === MISSING_ERROR_CODE
+        body.error.skip = options.ignoreMissing
+        && (body.error.code === MISSING_ERROR_CODE)
+        || body.error.code === NOT_SUPPORTED_CODE
```
Comment (alonweissfeld) [inline index.js]: Don't you mean 
```
var code = body.error.code
body.error.skip = options.ignoreMissing && (code === MISSING_ERROR_CODE || code === NOT_SUPPORTED_CODE)
```
(Note the parentheses)

Comment (alonbrody) [inline index.js]: Nope. The first two are in order to skip missing items so only if the `ignoreMissing` is `true` and the `code` is the same as `MISSING_ERROR_CODE` we will want to skip it. The new addition is separated from that.

Comment (alonweissfeld) [inline index.js]: Ahh ok. The would be understood more easily though with the right brackets placement
```
body.error.skip = (ignore && code === MISSING) || code === NOT_SUPPORTED
```

**Comment by alonbrody** [comment]
@alonweissfeld Fixed and added test. Good catch

**Pull request merged**